modules/afsmtp/afsmtp.c

```diff
@@ -134,6 +135,7 @@ afsmtp_dd_set_from(LogDriver *d, const gchar *phrase, const gchar *mbox)
 
   g_free(self->mail_from->phrase);
   g_free(self->mail_from->address);
+
   self->mail_from->phrase = afsmtp_wash_string(g_strdup(phrase));
```

Comment:
It's still not "my code" though, the only change was here the whitespace(which i will fix) BUT they are free-d somewhere at line 700 so that isn't a problem either.